src/main/java/io/github/thebusybiscuit/exoticgarden/listeners/PlantsListener.java

```diff
@@ -47,6 +49,8 @@
 import io.github.thebusybiscuit.slimefun4.libraries.dough.skins.PlayerSkin;
 import io.github.thebusybiscuit.slimefun4.libraries.paperlib.PaperLib;
 import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import io.github.thebusybiscuit.slimefun4.utils.tags.SlimefunTag;
+import static io.github.thebusybiscuit.slimefun4.utils.tags.SlimefunTag.values;
```

Comment:
This static import is not needed here...

Reply:
Tell that to the error i was having without static